### PR TITLE
made the text responsive

### PR DESCRIPTION
**Pull request by KaleemNeslit**
## Link to Issue
Closes: #9266 

## Description of Changes
- In FeatureHint.scss file  made below changes
- .FeatureHint change gap from 10 to 8
- .title-row added new flex-wrap: wrap

### CODE REVIEW

**Review by Israellund — APPROVED** [review]
LGTM

**Review by masvelio — APPROVED** [review]

**Comment by Israellund** [comment]
@KaleemNeslit At Common our protocol is that the engineer who made the PR merges the PR after it has 2 or more approvals. I'm not sure if I mentioned that to you. If you're unable to merge this PR for whatever reason please let me know and I can do it, otherwise feel free to merge this. Thank you.

**Pull request merged**
